api/src/main/java/org/openmrs/module/fhir/api/impl/DiagnosticReportServiceImpl.java

```diff
@@ -38,12 +41,37 @@
  */
 public class DiagnosticReportServiceImpl extends BaseOpenmrsService implements DiagnosticReportService {
 
+	public static final String FHIR_DIAGNOSTICREPORT_ORDER_TYPE_TO_HANDLER_MAP = "fhir.diagnosticreport.orderTypeToHandlerMap";
+
+	public static final String DEFAULT_HANDLER = "DEFAULT";
```

Comment:
I would suggest, change access modifier to private

Reply:
@corneliouzbett Thanks for the catch.